woocommerce/wc-admin
# Fix dependency declarations

The `dependency-group.js` custom eslint rule requires only Internal and External dependency blocks and requires Internal dependencies to have relative paths.

This PR removes `WooCommerce dependencies` block, changes aliased paths from the `client` folder to relative paths, and removes the webpack alias to the `client` folder.

### Detailed test instructions:

1. See the linter pass with 254 warnings and no errors.
2. Run `npm start` and ensure no errors are found in the logs.
2. Poke around the app looking for regressions. The likeliest of suspects would be a badly written relative path. I wrote a script to perform the task, so it should be OK.



### Changelog Note:

- Dev: Adhere to dependency-group rule from WooCommerce's eslint plugin.
